### DISCUSSION TITLE
sb Where did it come from?Can I rename it? I don't like this name

### DISCUSSION THREAD

**Discussion by luckyboy-wei** [seleniumbase/SeleniumBase#2568] [Q&A] [answered]
sb Where did it come from?Can I rename it? I don't like this name

![111](https://github.com/seleniumbase/SeleniumBase/assets/49641020/45d2e7f7-200e-487e-a5ab-b31eefd35a9e)

**Comment by mdmintz** [comment, marked as answer]
`sb` is the name of the SeleniumBase `pytest` fixture. See [help_docs/syntax_formats.md](https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/syntax_formats.md) for a list of all available Syntax Formats.
